.azure-build.yml

```diff
@@ -1,5 +1,13 @@
-queue:
-  name: Hosted VS2017
+strategy:
+  matrix:
+    Linux:
+      imageName: 'ubuntu-16.04'
+    Windows:
+      imageName: 'vs2017-win2016'
```

Comment:
Maybe we should go for `windows-2019`? Still in preview though...
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops

Reply:
I guess we can wait till it's out of preview. I'll be looking into running it on macos later on also, so we can return back to it as a part of the next PR.